althea_kernel_interface/src/ip_route.rs

```diff
@@ -4,6 +4,22 @@ use std::net::IpAddr;
 
 use failure::Error;
 
+pub enum IpRoute {
+    /// For creating default routes
+    DefaultRoute,
+    /// A route to a specific address
+    ToAddr(IpAddr),
+}
+
+impl ToString for IpRoute {
```

Comment:
not sure how I feel about the match self pattern but this is a nice style in general.